opae.spec.rhel

```diff
@@ -69,8 +72,7 @@ OPAE headers, tools, sample source, and documentation
 %setup -q -n %{name}-%{version}-%{opae_release}
 
 %build
-%cmake -DOPAE_MINIMAL_BUILD=ON \
-       .
+%cmake -DOPAE_MINIMAL_BUILD=ON 
```

Comment:
Isn't the . required?

Reply:
fedora-rawhide  fails mock -r fedora-rawhide-x86_64   --enable-network rebuild opae-2.1.0-1.fc34.src.rpm

CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_Fortran_FLAGS_RELEASE
    CMAKE_INSTALL_DO_STRIP
    INCLUDE_INSTALL_DIR
    LIB_INSTALL_DIR
    LIB_SUFFIX
    SHARE_INSTALL_PREFIX
    SYSCONF_INSTALL_DIR


-- Build files have been written to: /builddir/build/BUILD/opae-2.1.0-1
+ /usr/bin/cmake --build redhat-linux-build -j32 --verbose
Error: /builddir/build/BUILD/opae-2.1.0-1/redhat-linux-build is not a directory
error: Bad exit status from /var/tmp/rpm-tmp.OdYVXi (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.OdYVXi (%build)
Finish: rpmbuild opae-2.1.0-1.fc34.src.rpm
Finish: build phase for opae-2.1.0-1.fc34.src.rpm
ERROR: Exception(opae-2.1.0-1.fc34.src.rpm) Config(fedora-rawhide-x86_64) 0 minutes 41 seconds
INFO: Results and/or logs in: /var/lib/mock/fedora-rawhide-x86_64/result
ERROR: Command failed:

[Ananda] removed to "." to fix this fedora raw hide